packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -997,7 +1001,6 @@ export class CodeModelBuilder {
         }
       }
 
-      // track usage
       if (pollingSchema) {
```

Comment:
Did not remove this part because of this TCGC issue: https://github.com/Azure/typespec-azure/issues/1752